### DISCUSSION TITLE
Dexcom Share failure

### DISCUSSION THREAD

**Discussion by Navid200** [Navid200/xDrip#130] [Announcements]
There has been a change to the format used by the Dexcom servers resulting is xDrip failure to act as a follower.

A new version was released to address this:
https://github.com/NightscoutFoundation/xDrip/releases/tag/2021.12.02

**Comment by Navid200** [comment]
A new Nightly will be released shortly.
This release has not been tested as rigorously as usual because of the number of people affected by the current problem with the Dexcom share servers.
The intent is to get those systems back up and running.

If you experience any unexpected behavior after update, please explain the behavior in a post here:
https://github.com/Navid200/xDrip/discussions
Then, install the previous Nightly (December 1, 2021) to undo the update.  No need to uninstall first.

**Reply by Cheun1** [reply]
installed the trend arrow has returned and everything is ok for the moment, thank you very much.

**Reply by anglosid** [reply]
Is it not updated using the "Check for updates" option in xDrip+?

**Reply by Navid200** [reply]
@anglosid I have got several messages from you.  But, I cannot find the thread you have posted the messages in to respond to.
In response to this one, automatic update only is a notification.  It tells you there is an update and asks if you want to update.
If you confirm, it downloads the release.  Then, you will need to install it.

**Reply by anglosid** [reply]
Sorry, I was struggling to find the right thread. When I use the "check for updated version" xDrip does not present me with a new version. So, should I download it somewhere?

**Reply by Navid200** [reply]
This is how the xDrip update function works:
https://github.com/Navid200/xDrip/wiki/Updates

I have updated the first post of this thread.  It shows the release now.

**Reply by anglosid** [reply]
Thanks very much! I'm up and running again!

**Comment by aureliasychta** [comment]
#146

**Reply by Navid200** [reply]
@aureliasychta Would you please explain what you mean by your post?

We hope these discussions can help users and developers.  
The main objective of these discussions is to let users ask questions and receive guidance both from developers and other users.

The post you have linked to is an example.  If you like to offer help, you can post in it.
I have already contacted the developer who needs to attend to it and we have to be patient for him to respond.

This thread was meant to communicate an issue and how to address it.

If you need to reach me, you can send me a message:  https://github.com/Navid200
I welcome suggestions and feedback.

**Comment by aureliasychta** [comment]
Hi
I still have a problem
the new version did nothing

**Reply by Navid200** [reply]
Then, why have you posted here?
You should post in the thread you have linked and state that you have the same problem.
As I said, we are waiting for suggestions.
